hermes/src/api/sse.rs

```diff
@@ -99,16 +108,30 @@ pub async fn price_stream_sse_handler(
         async move {
             match message {
                 Ok(event) => {
+                    // Check if the event is out-of-order and if unordered updates are not allowed
+                    if let AggregationEvent::OutOfOrder { .. } = event {
+                        if !params.allow_unordered {
+                            return Ok(Event::default().comment("Out-of-order event skipped"));
+                        }
+                    }
```

Comment:
I think it's cleaner if the handle_aggregation_event does this as well. We want the business logic to be all in one place. Then you can say that `handle_aggregation_event` returns `Result<Option<PriceUpdate>>` to also be more clear when it is going to skip the events.

Reply:
agreed, will do